lambda/marchon.py

```diff
@@ -76,32 +94,62 @@ def get_sheet_data():
      16 Event Link, 17 Photo, 18 About
     '''
     # keep these fields
-    fields = {'name': 0, 'location': 1, 'contactName': 3, 'contactEmail': 5,
-              'facebook': 9, 'twitter': 10, 'instagram': 11, 'other': 12,
-              'website': 13, 'event': 14, 'eventDate': 15, 'eventLink': 16,
-              'photo': 17, 'about': 18}
+    fields = {
+        'name': 0,
+        'location': 1,
+        'contactName': 3,
+        'contactEmail': 5,
+        'facebook': 9,
+        'twitter': 10,
+        'instagram': 11,
+        'other': 12,
+        'website': 13,
+        'event': 14,
+        'eventDate': 15,
+        'eventLink': 16,
+        'photo': 17,
+        'about': 18
+    }
     return read_sheet('Sheet1!A1:S', fields, 1, True)
 
 
+def get_action_network_events() -> Dict:
+    log.info('\nstart get Action Network events')
+    return_events = get_events_from_events_campaign()
+    log.info('\ngot %d events from action network', len(return_events))
+    return return_events
+
+
 def get_geojson(url):
     log.info('\nload geojson')
     resp = requests.get('https://s3.amazonaws.com/ragtag-marchon/%s' % url)
     features = {}
     for feature in resp.json()['features']:
-        features[feature['properties']['location']] = feature
+        if feature['properties'].get('source', '') == 'actionnetwork':
```

Comment:
why is this special cased? actionnetwork events should return data in the same format

Reply:
Explained above.  This allows more than one action network event in the same location.

Reply:
@kielni perhaps one way to address this is to get rid of the special casing and use the same logic (allowing multiple events per location) for all events.

Reply:
ok that makes sense; maybe add a comment in the code so we can remember why they're different later without having to dig into another file?